tests/test_data/input_test_generate/scenario_C.json

```diff
@@ -259,8 +259,8 @@
         "factor": 0.001
       }
     },
-    "energy_feed_in": {
-      "example_pv_feedin": {
+    "local_generation": {
+      "example_local_generation": {
         "csv_file": "example_pv_feedin.csv",
```

Comment:
Please check all jsons in test-folder and adjust names:

1. remove "feedin"
2. remove project specific stuff like dates in building data, dhl etc.